lib/ansible/module_utils/scaleway.py

```diff
@@ -93,3 +93,32 @@ def update(self, path, data=None, headers=None):
     'ams1': {'name': 'Amsterdam 1', 'country': 'NL', "api_endpoint": 'https://cp-ams1.scaleway.com'},
     'EMEA-NL-EVS': {'name': 'Amsterdam 1', 'country': 'NL', "api_endpoint": 'https://cp-ams1.scaleway.com'}
 }
+
```

Comment:
A couple of concerns here.

1. Somehow we accepted this file without a license header.  This should be remedied immediately.  `module_utils` are typically licenses as BSD
1. A quick git blame indicates that you will also need permission from @maxamillion to relicense this code to `BSD`

Reply:
1. That was probably my fault to begin with, I was on a merge frenzy as the 2.6 community dev window was closing.
2. I consent to the relicense (do we need anything more formal for posterity than this GitHub comment?)

Reply:
This should be sufficient.

Reply:
@sieben please include the BSD license header in this module_utils file in this PR.

Reply:
I've updated the header with

```
# This file is part of Ansible
#
# Ansible is free software: you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation, either version 3 of the License, or
# (at your option) any later version.
#
# Ansible is distributed in the hope that it will be useful,
# but WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
# GNU General Public License for more details.
#
# You should have received a copy of the GNU General Public License
# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
```

Is that enough?

Reply:
It's not BSD but because this is the template that seem to be the most widespread I guess it makes sense to use it

Reply:
`module_utils` files should be BSD licensed.  Look in another `module_utils` file such as `basic.py` for a copy of the BSD license to be included.

Reply:
done